libraries/GCS_MAVLink/GCS.h

```diff
@@ -772,6 +772,9 @@ class GCS
     // update uart pass-thru
     void update_passthru();
 
+    static uint16_t sys_status_errors1();
+    static uint16_t sys_status_errors2();
```

Comment:
Unused?

Reply:
Removed.  I added it as a hint as to what was going on.